pipelines/matrix/src/matrix/pipelines/modelling/preprocessing_model.py

```diff
@@ -0,0 +1,41 @@
+import numpy as np
```

Comment:
This indeed solves a problem we had before, where it's not possible to grab a model and try to use it without applying its transformers.